settings.go

```diff
@@ -258,10 +268,17 @@ func (st *Settings) SetAck(ack bool) {
 }
 
 func (st *Settings) Deserialize(fr *FrameHeader) error {
+	if len(fr.payload)%6 != 0 {
```

Comment:
I prefer to remove this check and throw an error when we process the settings. What do you think?

Reply:
For me it's like an "invalid frame", so I prefer it here, but I can change it.

Reply:
I'll leave it up to you